features/environment.py

```diff
@@ -130,6 +131,11 @@ def _install_project_requirements(context):
         .splitlines()
     )
     install_reqs = [req for req in install_reqs if "{" not in req and "#" not in req]
-    install_reqs.append("kedro-datasets[pandas-csvdataset]")
+    # For Python versions 3.9 and above we use the new dataset dependency format introduced in `kedro-datasets` 3.0.0
+    if sys.version_info.minor > 8:  # noqa: PLR2004
```

Comment:
Shall we use a constant here to increase readability a bit?

Reply:
Done!